lib/metrics/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const MetricsRegistry = require('./registry')
+
+const registrySymbol = Symbol('metrics-registry')
+const agentSymbol = Symbol('metrics-agent')
+
+class Metrics {
+  constructor (agent) {
+    this[agentSymbol] = agent
+    this[registrySymbol] = undefined
```

Comment:
I normally use `null` instead of `undefined` in these situations, but that's based on the idea that it's better for the V8 optimizer. Not sure if that's true any more?